style.css

```diff
@@ -152,21 +154,156 @@ button:focus {
   width: 218px;
 }
 
-.card-back:hover {
-  transform: translate3d(-10px, -30px, -40px) rotate(-3.08deg);
-  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
-  transition: .8s;
-  }
-
 .card-back,
 .card-front {
   position: absolute;
   -webkit-backface-visibility: hidden;
   backface-visibility: hidden;
 }
 
+.card-back:hover {
+  transform: translate3d(-10px, -30px, -40px) rotate(-3.08deg);
+  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
+  transition: .8s;
+}
+
 .card-front {
   transform: rotateY(180deg);
 }
 
+@media (max-width: 980px) {
+  .title {
+    font-size: 36px;
+  }
+
+  .text {
+    font-size: 12px;     
+  }
+
+  .difficults {
+    font-size: 32px;
+    line-height: 34px;
+    padding-left: 119px;    
+  }
+
+  .card-wrap {
+    width: 38%;
+  }
+
+  .start-card {
+    width: 100%;
+  }
+
+  .difficult-check {
+    top: 36px;
+  }
+
+  .difficult:nth-child(1) .difficult-check {
+    top: 10px;
+  }
+}
+
+@media (max-width: 710px) {
+  .title {
+    padding-left: 60px;
+  }
+
+  .text {
+    padding-left: 65px;
+  }
+
+  .difficults {
+    padding-left: 80px;
+    line-height: 32px;
+  }
+
+  .btn {
+    margin-left: 48px;
+  }
+
+  .cards {
+    margin-right: 0;
+  }
+}
+
+@media (max-width: 675px) {
+  .bottom {
+    flex-flow: wrap;
+    margin-top: 53px;  
+  }
 
+  .cards {
+    margin-left: 53px;
+  }
+}
+
+@media (max-width: 610px) {
+  .title {
+    padding-left: 25px;
+  }
+
+  .text {
+    padding-left: 32px;
+  }
+
+  .difficults {
+    padding-left: 37px;
+  }
+
+  .btn {
+    margin-left: 29px;
+  }    
+    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/jsFinalTask/pullRequest?prid=6141138)